docs/api/components/schemas.yaml

```diff
@@ -1878,18 +1878,28 @@ Database-audit:
   title: Database-audit
   description: A map of audit events and whether they are enabled or not.
   type: object
-  additionalProperties:
-    x-additionalPropertiesName: audit_id
-    description: The audit event ID and whether it is enabled or not.
-    type: boolean
+  properties:
+    enabled:
+      type: boolean
+    events:
+      type: object
+      additionalProperties:
+        x-additionalPropertiesName: audit_id
+        description: The audit event ID and whether it is enabled or not.
+        type: boolean
```

Comment:
Is this worth adding an example for the format? I am somewhat confused by why this is `map[string]any` and then it is actually map[string]bool?

Reply:
The `PUT`/`POST` endpoint accepts both verbose and non-verbose formats (without specifying the `?verbose=true` query param)

This is displayed correctly for both cases in the Redocly UI with a dropdown to select the type of body format.